source/MaterialXView/Viewer.h

```diff
@@ -146,6 +182,8 @@ class Viewer : public ng::Screen
         _exitRequested = true;
     }
 
+    void bakeTextures();
```

Comment:
Unintended reordering of variable declarations?

Reply:
Required to move from private to protected.

Reply:
Got it, and this makes sense.